lib/rules/forbid-prop-types.js

```diff
@@ -103,7 +125,7 @@ module.exports = {
         } else if (value.type === 'Identifier') {
           target = value.name;
         }
-        if (isForbidden(target)) {
+        if (isForbidden(target, value)) {
           context.report({
             node: declaration,
             message: `Prop type \`${target}\` is forbidden`
```

Comment:
Would it be worth adding a new error message here?

Reply:
yes - when the `empty` option is set, I think the message should be more specific.